test/multifile_samples/import_rewrites/instrumentationMetadata.json.expected

```diff
@@ -0,0 +1,125 @@
+{
+  "instrToOriginalMap": [
+    [
+      "25:33:0",
+      "27:33:0"
+    ],
+    [
+      "46:9:0",
+      "48:9:0"
+    ],
+    [
+      "46:7:0",
+      "48:7:0"
+    ],
+    [
+      "59:22:0",
+      "27:22:1"
+    ],
+    [
+      "72:1:0",
+      "40:1:1"
+    ],
+    [
+      "75:1:0",
+      "43:1:1"
+    ],
+    [
+      "78:1:0",
+      "46:1:1"
+    ],
+    [
+      "83:22:0",
+      "51:22:1"
+    ],
+    [
+      "96:1:0",
+      "64:1:1"
+    ],
+    [
+      "99:1:0",
+      "67:1:1"
+    ],
+    [
+      "102:1:0",
+      "70:1:1"
+    ],
+    [
+      "107:26:0",
+      "75:24:1"
+    ],
+    [
+      "124:1:0",
+      "90:1:1"
+    ],
+    [
+      "127:1:0",
+      "93:1:1"
+    ],
+    [
+      "130:1:0",
+      "96:1:1"
+    ],
+    [
+      "263:16:0",
+      "166:16:2"
+    ],
+    [
+      "280:121:0",
+      "48:83:3"
+    ],
+    [
+      "296:4:0",
+      "64:4:3"
+    ],
+    [
+      "296:4:0",
+      "64:4:3"
+    ],
+    [
+      "307:18:0",
+      "75:9:3"
+    ],
+    [
+      "307:7:0",
+      "75:7:3"
+    ],
+    [
+      "331:17:0",
+      "90:8:3"
+    ],
+    [
+      "331:5:0",
+      "90:5:3"
+    ],
+    [
+      "354:17:0",
+      "104:10:3"
+    ],
+    [
+      "354:5:0",
+      "104:7:3"
+    ],
+    [
+      "377:21:0",
+      "120:8:3"
+    ],
+    [
+      "377:9:0",
+      "120:5:3"
+    ]
+  ],
+  "otherInstrumentation": [
+    "221:38:0"
+  ],
+  "propertyMap": [],
+  "originalSourceList": [
+    "test/multifile_samples/import_rewrites/imp2.sol",
+    "test/multifile_samples/import_rewrites/imp3.sol",
+    "test/multifile_samples/import_rewrites/imp1.sol",
+    "test/multifile_samples/import_rewrites/main.sol"
+  ],
+  "instrSourceList": [
+    "--"
```

Comment:
Is that intentional?

Reply:
Yes. The "--" indicates that the output was written to stdout.